bash_tests/language_translator_1.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# The following will translate two phrases, "Hello, world!" and "How are you?" from English to Spanish.
+
+echo "Translate two phrases from English to Spanish"
+
+if test "$#" -ne 1; then
+    echo "Usage: $0 api-key"
```

Comment:
should we use `apikey` in the help string to be consistent with what Watson services expect in the POST header? (also applies to all the other scripts…)

Reply:
Agreed. I will update it